docs/03-tutorials/00-api-exposure/apix-03-own-domain.md

```diff
@@ -153,47 +66,13 @@ Follow these steps to set up your custom domain and prepare a certificate requir
   EOF
   ```
 
-  </details>
-  <details>
-  <summary>
-  Non-Gardener cluster
-  </summary>
-
-  ```bash
-  cat <<EOF | kubectl apply -f -
-  apiVersion: dns.gardener.cloud/v1alpha1
-  kind: DNSProvider
-  metadata:
-    name: dns-provider
-    namespace: $NAMESPACE
-  spec:
-    type: $SPEC_TYPE
-    secretRef:
-      name: $SECRET
-    domains:
-      include:
-        - $DOMAIN
-  EOF
-  ```
-
-  </details>
-  </div>
-
-- Create a DNSEntry CR. Export values of the **metadata.spec.dnsName**, and **metadata.spec.targets.IP** parameters as environment variables. Optionally, you can also change the value of **metadata.spec.ttl**. Run:
+- Export the following values as environment variables and run the command provided:
```

Comment:
I think we need 3 (?) blank spaces for proper indentation here. Also for the sample code and Note box below.